cmake/modules/FindGLEW.cmake

```diff
@@ -62,6 +62,12 @@ if (WIN32)
         PATH_SUFFIXES
             Release/${ARCH}
         DOC "The GLEW library")
+
+    if ("${GLEW_LIBRARY}" MATCHES "glew32s(d|)")
+        add_definitions("/DGLEW_STATIC")
```

Comment:
Yeah, that's why build-club is using dynamic now. Yesterday, I did a build of Pixar's dev_win_ip with @JamieKenyonFoundry's latest dev_win_ip merged on top, and usd-build-club built GLEW and OpenSubdiv and it all worked :) What link problem are you seeing in OSD?